R/annotation-logticks.r

```diff
@@ -200,7 +200,10 @@ GeomLogticks <- ggproto("GeomLogticks", Geom,
     gTree(children = do.call("gList", ticks))
   },
 
-  default_aes = aes(colour = "black", size = 0.5, linetype = 1, alpha = 1)
+  default_aes = expr(aes(
```

Comment:
I don't think you need the `expr()` wrapper here. `aes()` already captures the unevaluated aesthetics?

Reply:
It does, as quosures, so iirc, without `expr()` I ended up needing to do some quosure hacking -- to set the environment to one that understood `theme`. As I refactor the theme evaluation into it's own function as you suggest above, I'll aim to eliminate this as well.